client/state/posts/selectors.js

```diff
@@ -359,6 +360,29 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export const editedPostHasContent = createSelector(
+	( state, siteId, postId ) => {
+		const editedPost = getEditedPost( state, siteId, postId );
+		return (
+			!! editedPost &&
+			(
+				some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
+				! isPostContentEmpty( editedPost.content )
+			)
+		);
+	},
+	( state ) => [ state.posts.items, state.posts.edits ]
+);
```

Comment:
Here are the results:

**No memoization**
`1.3ms` for `getEditedPost` and `0.5ms` for `editorPostHasContent`
<img width="1079" alt="capture d ecran 2016-11-11 a 8 54 46 am" src="https://cloud.githubusercontent.com/assets/272444/20208554/2a47efe2-a7ef-11e6-9646-b7e60c0fdf5d.png">
<img width="1080" alt="capture d ecran 2016-11-11 a 8 55 47 am" src="https://cloud.githubusercontent.com/assets/272444/20208561/3284f1a0-a7ef-11e6-8e2d-49bab087a7a0.png">

**Only getEditedPost is memoized**
`~0ms` for `getEditedPost` and `0.4ms` for `editorPostHasContent`
<img width="1082" alt="capture d ecran 2016-11-11 a 9 06 17 am" src="https://cloud.githubusercontent.com/assets/272444/20208562/3288c384-a7ef-11e6-8f6c-2570b1bee15e.png">

**the two selectors memoized**
`~0ms` for the two selectors

(I don't have screenshots when I say `~0ms` because the results are too low, Google Chrome does not show them for these functions)

Thoughts?